sdk/appconfiguration/app-configuration/README.md

```diff
@@ -0,0 +1,106 @@
+## Azure ConfigurationClient SDK for JavaScript
+
+This package contains an isomorphic SDK for ConfigurationClient.
+
+### Currently supported environments
+
+- Node.js version 6.x.x or higher
```

Comment:
Is the README configurable at all through autorest? I think we're now saying we only support version 8.x.x or higher, and should be using `@azure/identity` over `@azure/ms-rest-nodeauth` if possible.

Since this package also includes the convenience layer we should probably make these changes (though they can probably be done in a separate PR to get this base out), but will it get replaced if we need to regenerate from the models again?

Reply:
When generating code, there is an option to skip the readme